src/main/java/io/stargate/sgv2/jsonapi/service/resolver/CommandResolver.java

```diff
@@ -150,10 +151,15 @@ default void addToMetrics(
       Command command,
       LogicalExpression logicalExpression,
       IndexUsage baseIndexUsage) {
-    // TODO: this functions hould not be on the CommandResolver interface, it has nothing to do with
+    // TODO: this functions should not be on the CommandResolver interface, it has nothing to do
+    // with
     // that
     // it's only here because of the use of records and interfaces, move to a base class
-    Tag commandTag = Tag.of(jsonApiMetricsConfig.command(), command.getClass().getSimpleName());
+    Tag commandTag =
+        Tag.of(
+            jsonApiMetricsConfig.command(),
+            DeprecatedCommandUtil.maybeResolveDeprecatedCommandName(
```

Comment:
Could resolve actual command `class` instead of name?

Reply:
solved, thanks